connect.js

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types';
 import canReflect from 'can-reflect';
 import ObservableComponent from './observable-component';
 
+//!steal-remove-start
+console.log(React.version)
+//!steal-remove-end
+
```

Comment:
Was this a mistake left in?

Reply:
no, its something that nedes ot be fixed =/